News: announce community meeting in June (#359)
OSGeo/grass-website: content/news/2023_04_08_GRASS_Community_meeting_Prague_june_2023.md

---
title: "GRASS GIS Community meeting in Prague!"
date: 2023-04-07T23:12:41+02:00
layout: "news"
author: Veronica Andreo and the GRASS Development Team
---

#### We are organizing a **Community Meeting** to celebrate GRASS GIS 40th birthday!! 

<a href="/images/news/grass_sprint2018_bonn_fotowall_medium.jpg">
  <img src="/images/news/grass_sprint2018_bonn_fotowall_medium.jpg"
   alt="Bonn 2018 Community Sprint"
   title="Bonn 2018 Community Sprint"
   width="65%" style="float:right;padding-left:25px;padding-top:15px">
</a>

The [GRASS GIS Community Meeting](https://grasswiki.osgeo.org/wiki/GRASS_Community_Sprint_Prague_2023) 
with users, supporters, contributors, power users and developers 

> will take place from June 2 to 6, 2023, at the 

Reviewer: ```suggestion
will take place from **June 2 to 6, 2023**, at the 
```

[Faculty of Civil Engineering](https://www.fsv.cvut.cz/?lang=en) 
from the [Czech Technical University](https://www.cvut.cz/en) in Prague, Czech Republic.

> Community meetings are great occasions to support GRASS GIS development by actively 
> contributing to the source code, documentation, translations, website or likewise. 
> The community meeting is also a get-together where supporters, contributors, power 
> users and developers make decisions and tackle larger problems related to the project, discuss 
> and collaboratively resolve bugs, plan the direction for the project and work on new features. 
> We welcome people committed to improving the GRASS GIS interfaces to QGIS, GDAL, 
> PostGIS, R statistics, OGC Services and willing to celebrate with us the 
> **40th GRASS GIS birthday!!**

Reviewer: ```suggestion
Community meetings are great opportunities to support the development of GRASS GIS by 
actively contributing to the source code, documentation, translations, website or likewise. 
The community meeting is also a get-together where supporters, contributors, power 
users and developers make decisions and tackle larger problems related to the project, discuss 
and collaboratively resolve bugs, plan the direction of the project and work on new features. 
We welcome people committed to improving the GRASS GIS interfaces to QGIS, GDAL, 
PostGIS, R statistics, OGC services and willing to celebrate with us the 
**40th birthday of GRASS GIS!!**
```

> To support as many attendants as possible, especially students and developers with far arrival, 
> **we welcome sponsors and financial contributions** via [OpenCollective](https://opencollective.com/osgeo/projects/grass).

Reviewer: ```suggestion
To support as many attendants as possible, especially students and developers with from far away, 
**we welcome sponsorship and financial contributions** via [OpenCollective](https://opencollective.com/osgeo/projects/grass).
```

Your contribution counts!

<br>
<div align="center">
<button class="btn btn-primary">
<b><a href="https://opencollective.com/grass" target="_blank">Support GRASS GIS Community Meeting</a></b>
</button>
</div>

<br>

> Keep an eye at this wiki for updates and details on the agenda:

Reviewer: ```suggestion
Keep an eye on this wiki page for updates and agenda details:
```

<https://grasswiki.osgeo.org/wiki/GRASS_Community_Sprint_Prague_2023>. 

> Everyone is welcome to participate and support us with donations!

Reviewer: ```suggestion
Everyone is welcome to participate and support us by donating!!
```
